presto-docs/src/main/sphinx/release/release-0.170.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.170
+=============
+
+General Changes
+---------------
+
+* Fix broken Web UI rendering when catalog properties are set.
+* Fix rendering of live plan when query is queued.
+* Fix race condition that could cause queries to fail with InterruptedException in rare cases.
+* Fix a performance regression for ``GROUP BY`` queries over ``UNION``.
+* Fix performance regression when a significant number of exchange sources produce
```

Comment:
This looks strange compared to the line above since it's lacking the "a". Maybe we could phrase as "Fix a performance regression that occurs when ..."
```
* Fix a performance regression that occurs when a significant number of exchange
  sources produce no data during an exchange (e.g., in a skewed hash join).